### PR TITLE
Add page_size query parameter

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3915](https://openedx.atlassian.net/browse/DOC-3915)

Added the page_size query parameter to the enterprise-catalogs/{catalog_id} Endpoint section

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @mattdrayer 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Squash commits

### CODE REVIEW

**Review by mattdrayer — APPROVED** [review]
Thanks for the quick turnaround!

File: en_us/enterprise_api/source/api_reference/reference.rst [hunk 1]
```diff
@@ -100,6 +101,21 @@ Example Request
    https://api.edx.org/enterprise/v1/enterprise-catalogs/1/4g1BB0us321/ \
    -H "Authorization: JWT {access token}"
 
+=====================
+Parameters
+=====================
+
+You can use an optional ``page_size`` parameter to specify the number of
+content items that ``enterprise-catalogs/{catalog_id}`` calls should return
```
Comment (srpearce) [inline en_us/enterprise_api/source/api_reference/reference.rst]: Should or must?

File: en_us/enterprise_api/source/api_reference/reference.rst [hunk 2]
```diff
@@ -100,6 +101,21 @@ Example Request
...
+on each page of the response. If the ``page_size`` parameter is not supplied,
+the ``enterprise-catalogs/{catalog_id}`` call returns the default value of 20 content items per page. For example:
```
Comment (srpearce) [inline en_us/enterprise_api/source/api_reference/reference.rst]: Line wrap

File: en_us/enterprise_api/source/api_reference/reference.rst [hunk 3]
```diff
@@ -100,6 +101,21 @@ Example Request
    https://api.edx.org/enterprise/v1/enterprise-catalogs/1/4g1BB0us321/ \
    -H "Authorization: JWT {access token}"
 
+=====================
+Parameters
+=====================
+
+You can use an optional ``page_size`` parameter to specify the number of
+content items that ``enterprise-catalogs/{catalog_id}`` calls should return
+on each page of the response. If the ``page_size`` parameter is not supplied,
```
Comment (srpearce) [inline en_us/enterprise_api/source/api_reference/reference.rst]: "If the (call? user?) does not supply the ``page_size`` parameter"

**Review by srpearce — COMMENTED** [review]
👍 modulo minor comments

**Pull request merged**